### PR TITLE
Bonnie 1604 merge communication

### PR DESCRIPTION
**Pull request by edeyoung**
QDM 5.4 merges communication types and adds attributes.

Communication, Provider to Patient
Communication, Patient to Provider
Communication, Provider to Provider

becomes - Communication, Performed

Attributes for the DataType:
• Category
• Sender
• Recipient
• Medium
• Negation Rationale (already present)
• Relevant Period (already present)
• Author dateTime (already present)
• relatedTo (already present)
• Code (already present)
• id (already present)

Pull requests into Health Data Standards require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.
 
**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1671
- [x] Internal ticket links back to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] ~Tests are included and test edge cases.~ can't effectively test w/o a MAT export that uses this. See https://jira.mitre.org/browse/BONNIE-1674.
- [x] Tests have been run locally and pass
- [x] Code coverage has not gone down and all code touched or added is covered.
     * bonnie-v3.0: 90.49
     * this branch: 90.49
 
**Cypress Reviewer:**
 
Name: dczulada
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code
 
**Bonnie Reviewer:**
 
Name: @losborne
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: lib/health-data-standards/models/communication.rb [hunk 1]
```diff
@@ -1,3 +1,13 @@
 class Communication < Entry
   field :direction,           type: String
```
Comment (lukoktonos) [inline lib/health-data-standards/models/communication.rb]: Should `:direction` be removed from Communication?  There is also in `/import/cda/communication_importer.rb`
```
  17           communication.direction = find_communication_direction(entry_element)
...
  25         def find_communication_direction(entry_element)
  26           case entry_element.at_xpath(@template_xpath).value
  27           when "2.16.840.1.113883.10.20.24.3.3"
  28             return "communication_from_provider_to_patient"
  29           when "2.16.840.1.113883.10.20.24.3.2"
  30             return "communication_from_patient_to_provider"
  31           when "2.16.840.1.113883.10.20.24.3.4"
  32             return "communication_from_provider_to_provider"
  33           end
  34         end
  35 

```

Comment (lukoktonos) [reply lib/health-data-standards/models/communication.rb]: Talked to @jbradl11 and added comments indicating these are deprecated as of QDM 5.4

**Review by lukoktonos — APPROVED** [review]

**Review by dczulada — APPROVED** [review]
